client.sy

```diff
@@ -4,6 +4,14 @@ start :: fn {
     game.bind_keys'
 
     n_rpc_connect' "127.0.0.1"
+    n_rpc_server' game.player_join
```

Comment:
Maybe we could add a library function that blocks until we're connected to the server here?

Reply:
Then we can't close the game? I think it's better to have a much control as possible in the language. Maybe a callback is a more flexible idea?